Classes/Emogrifier.php

```diff
@@ -298,7 +306,17 @@ public function emogrify()
 
         $this->copyCssWithMediaToStyleNode($cssParts, $xmlDocument);
 
-        return $xmlDocument->saveHTML();
+        if ($this->returnBodyContentOnly) {
```

Comment:
Instead of having a behavior switch (which is a bad smell), I'd prefer a separate method named something like "emogrifyBody" or "emogrifyBodyOnly".